setup.py

```diff
@@ -1,73 +0,0 @@
-#!/usr/bin/env python
```

Comment:
the one minor downside to this is that we are likely to lose the "[used by](https://github.com/wagtail/wagtail-localize/network/dependents?package_id=UGFja2FnZS00NDI2MDYwNzk%3D)" GitHub feature, but we can live with it